### PR TITLE
Connpass (Independent Publisher)

### PR DESCRIPTION
**Pull request by Miyake-Mito**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image

image

### CODE REVIEW

**Comment by v-EgorKozhadei** [comment]
Hello @Miyake-Mito, 

Thank you very much and congratulations on your new submission. Our Certification Engineers will review it within 1-2 weeks. 

Please let us know if you have any questions. 

Thank you very much again for working with us.

File: independent-publisher-connectors/Connpass/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,322 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "connpass",
+    "description": "Using the connpass API, you can search and collect information about IT workshops and events.",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Connpass/apiDefinition.swagger.json:5]: @Miyake-Mito please remove the word API or replace it with something like 'service'. Using the word API is not allowed in the description.

File: independent-publisher-connectors/Connpass/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "results_start": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "results_start",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Connpass/apiDefinition.swagger.json:31]: @Miyake-Mito please look through all descriptions and titles and make sure that they look like valid sentences. More information about English strings can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: independent-publisher-connectors/Connpass/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+                      }
+                    }
+                  },
+                  "description": "events"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get_event",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Connpass/apiDefinition.swagger.json:186]: @Miyake-Mito please change it to `Get event`. More information about summary English strings [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: independent-publisher-connectors/Connpass/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+        "operationId": "Get_event",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Connpass/apiDefinition.swagger.json:187]: @Miyake-Mito please use pascal case for operationId properties. You can [refer to coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) for more information and examples.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @Miyake-Mito,

Please check a few comments after the review.

Let us know if you have any questions.

File: independent-publisher-connectors/Connpass/apiDefinition.swagger.json (hunk 1 above)
Comment (Miyake-Mito) [reply independent-publisher-connectors/Connpass/apiDefinition.swagger.json]: レビューしてくれてありがとうございます。
'API' の文字を消しました。

File: independent-publisher-connectors/Connpass/apiDefinition.swagger.json (hunk 2 above)
Comment (Miyake-Mito) [reply independent-publisher-connectors/Connpass/apiDefinition.swagger.json]: Thank you for reviewing this.
All descriptions and titles have been checked and corrected.

File: independent-publisher-connectors/Connpass/apiDefinition.swagger.json (hunk 3 above)
Comment (Miyake-Mito) [reply independent-publisher-connectors/Connpass/apiDefinition.swagger.json]: Thank you for reviewing this.
Changed summary to "Search Events".

File: independent-publisher-connectors/Connpass/apiDefinition.swagger.json (hunk 4 above)
Comment (Miyake-Mito) [reply independent-publisher-connectors/Connpass/apiDefinition.swagger.json]: Thank you for reviewing this.
I have applied pascal case to the operationId properties.

**Comment by v-EgorKozhadei** [comment]
[[certify-connector]]

**Comment by v-EgorKozhadei** [comment]
Hello @Miyake-Mito, 

I hope you are doing well. 

Congratulations, your connector is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.  

Please let us know if you have any questions. 

Thank you very much for working with us.

**Comment by Miyake-Mito** [comment]
コネクタを承認していただきありがとう。
コネクタが展開されるのを楽しみにしています。

**Comment by Miyake-Mito** [comment]
@v-EgorKozhadei 
Thanks for approving the connector.
We look forward to seeing the connector rolled out.

**Comment by Miyake-Mito** [comment]
Hello @v-EgorKozhadei ,
Thanks for approving the connector.
We look forward to seeing the connector rolled out.

**Comment by Amjed-Ayoub** [comment]
Hello @Miyake-Mito,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@Miyake-Mito Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**